src/widget/number-input/number-input.js

```diff
@@ -34,6 +34,7 @@ class NumberInput extends Widget {
 
         const question = input.closest('.question');
 
+        // analog-scale is included as a courtesy to OpenClinica
```

Comment:
This (and others) seems fine for now, but I'd like to think about what an extensible interface for these cases might look like. Partly because these will become harder to track and account for with the OpenClinica code removed (i.e. I only knew to add this here by looking at other usage of number inputs). But mainly because CSS selectors and similar DOM state lookups are _by far_ our biggest performance challenge in Core.

Reply:
> but I'd like to think about what an extensible interface for these cases might look like

Yes, would definitely be best. 

Thanks!